bower.json

```diff
@@ -3,21 +3,22 @@
   "description": "EAO Project Space",
   "version": "0.4.1",
   "dependencies": {
-    "angular": "latest",
-    "angular-animate": "~1.5",
-    "angular-bootstrap": "~2.5.0",
-    "angular-bootstrap-confirm": "~0.7.0",
-    "angular-cookies": "~1.4.0",
-    "angular-drag-and-drop-lists": "^2.1.0",
-    "angular-file-upload": "^2.4.0",
-    "angular-google-maps": "latest",
-    "angular-messages": "~1.5",
-    "angular-mocks": "~1.5",
-    "angular-moment": "latest",
-    "angular-resizable": "latest",
-    "angular-resource": "~1.5",
-    "angular-scroll": "~1.0.0",
-    "angular-smart-table": "2.1.8",
+    "angular": "1.6.9",
```

Comment:
Updated to use 1.6.9 rather than latest.  Though possible unnecessary as 1.6.9 is the last version ever? and so is always latest?

Reply:
Yeah, not sure. I guess pinning the dependency is good either way.